app/Config/Paths.php

```diff
@@ -75,4 +75,14 @@ class Paths
      * is used when no value is provided to `Services::renderer()`.
      */
     public string $viewDirectory = __DIR__ . '/../Views';
+
+    /**
+     * ---------------------------------------------------------------
+     * ENVIRONMENT DIRECTORY NAME
+     * ---------------------------------------------------------------
+     *
+     * This variable must contain the name of the directory for
+     * environment files.
+     */
+    public string $environmentDirectory = __DIR__ . '/../../';
```

Comment:
```suggestion
    public string $envDirectory = __DIR__ . '/../../';
```

Can we rename this property to `$envDirectory`?   

This would make the purpose clearer, since "environment" is mostly used in the context of the `CI_ENVIRONMENT` constant (as also shown in the user guide: https://codeigniter.com/user_guide/general/environments.html). Using `envDirectory` helps avoid confusion and more accurately reflects that this refers to the `.env` file, not the runtime environment.